src/pcapi/routes/pro/venue_labels.py

```diff
@@ -1,14 +1,19 @@
-from flask import jsonify
 from flask_login import login_required
 
+from pcapi.core.offerers import repository as offerers_repository
 from pcapi.flask_app import private_api
-from pcapi.infrastructure.container import get_venue_labels
-from pcapi.routes.serialization.venue_labels_serialize import serialize_venue_label
+from pcapi.routes.serialization.venue_labels_serialize import VenueLabelListResponseModel
+from pcapi.routes.serialization.venue_labels_serialize import VenueLabelResponseModel
+from pcapi.serialization.decorator import spectree_serialize
 
 
-# @debt api-migration
 @private_api.route("/venue-labels", methods=["GET"])
 @login_required
-def fetch_venue_labels():
-    venue_labels = get_venue_labels.execute()
-    return jsonify([serialize_venue_label(venue_label) for venue_label in venue_labels]), 200
+@spectree_serialize(response_model=VenueLabelListResponseModel)
+def fetch_venue_labels() -> VenueLabelListResponseModel:
+    venue_label_orm_models = offerers_repository.get_all_venue_labels()
+    venue_label_response_models = [
+        VenueLabelResponseModel.from_orm(venue_label) for venue_label in venue_label_orm_models
+    ]
+
+    return VenueLabelListResponseModel(__root__=venue_label_response_models)
```

Comment:
(détail / style)

Suggestion pour une version un peu plus concise, avec un peu moins de variable intermédiaire avec_des_noms_à_rallonge :

```suggestion
    labels = [
        VenueLabelResponseModel.from_orm(venue_label)
        for venue_label in offerers_repository.get_all_venue_labels()
    ]
    return VenueLabelListResponseModel(__root__=labels)
```